tests/snmpsim/ray.snmprec

```diff
@@ -3,8 +3,32 @@
 1.3.6.1.2.1.1.2|6|1.3.6.1.4.1.33555.1.1
```

Comment:
Create a new snmprec file (the -v option), you've merged data for two different types of devices...